### ISSUE TITLE
Build fails: no member named 'setGlyphImageMargin' in 'osgText::Font'

### ISSUE THREAD

**Issue by yurivict** [gwaldron/osgearth#1022]
```
src/osgEarth/Registry.cpp:144:23: error: no member named 'setGlyphImageMargin' in 'osgText::Font'
        _defaultFont->setGlyphImageMargin( 2 );
        ~~~~~~~~~~~~  ^
src/osgEarth/Registry.cpp:697:26: error: no member named 'setThreadSafeReferenceCounting' in 'osg::Referenced'
        osg::Referenced::setThreadSafeReferenceCounting( true );
        ~~~~~~~~~~~~~~~~~^
```

osgearth-2.8
osg-3.5.9

**Comment by remoe** [comment]
OSG 3.5.9 was not available when osgEarth 2.8 was released. Try the latest master of osgEarth or use an older OSG version.

**Comment by gwaldron** [comment]
fixed
